ur_robot_driver/resources/ursim_driver/README.md

```diff
@@ -0,0 +1,21 @@
+# Docker support for Universal Robots ROS2 Driver and URsim
+
+## Prerequisites
+- Download [vncviewer](https://www.realvnc.com/en/connect/download/viewer/) from the official site.
+- Install [Docker Engine](https://docs.docker.com/engine/install/ubuntu/) and [Docker Compose](https://docs.docker.com/compose/install/).
+
+## Driver and URsim
+Driver and URsim can be run together by following command  which will build necessary docker images and start the containers.
+With appropriate arguments set the ROS2 distro and the Moveit2! build is required.
+```
+ROS2_DISTRO=foxy BUILD_MOVEIT2=0 docker-compose up --build
```

Comment:
Should we separate somehow build and execution. First run takes quite a few time and than at some point everything pops up. Would it be possible/sensible to notify user somehow when build finishes.